cms/templates/sudo/sudo.html

```diff
@@ -1,7 +1,8 @@
 {% block body %}
```

Comment:
Hrm... My response to that comment makes it seem like I was in favor of a django version as well...